railties/lib/rails/generators/rails/authentication/templates/app/controllers/concerns/authentication.rb.tt

```diff
@@ -21,16 +21,14 @@ module Authentication
       resume_session || request_authentication
     end
 
-
     def resume_session
       Current.session ||= find_session_by_cookie
     end
 
     def find_session_by_cookie
-      Session.find_by(id: cookies.signed[:session_id])
+      Session.find_by(id: cookies.signed[:session_id]) if cookies.signed[:session_id]
```

Comment:
Tangential, but I'm surprised rails issues a query for nil primary keys

Reply:
`find_by` doesn't expect queries searching by primary keys but any field. Worth checking if `find_by(field: nil)` would translate to `WHERE field IS NULL`, which is a valid and useful case.